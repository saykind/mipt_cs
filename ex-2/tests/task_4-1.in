qwe 123456789 3
