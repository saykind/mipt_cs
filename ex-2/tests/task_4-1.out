12qwe3456789